modules/3d/include/opencv2/3d/volume_settings.hpp

```diff
@@ -6,13 +6,14 @@
 #ifndef OPENCV_3D_VOLUME_SETTINGS_HPP
 #define OPENCV_3D_VOLUME_SETTINGS_HPP
 
+
 #include <opencv2/core.hpp>
 #include <opencv2/3d/volume.hpp>
 
 namespace cv
 {
 
-enum class VolumeType
+enum VolumeType
```

Comment:
Why is "class" dropped?
In that case we should have appropriate prefix for values names.

Reply:
dropped for correct python bindings

Reply:
I believe class should not dropped ...

Reply:
@DumDereDum This item is not resolved yet.

Reply:
@alalek after adding `class` I got error
```
======================================================================
ERROR: test_VolumeTSDF (test_volume.volume_test)
----------------------------------------------------------------------
Traceback (most recent call last):
File ".../opencv/modules/3d/misc/python/test/test_volume.py, line 33, in test_VolumeTSDF
volume = cv.Volume(cv.TSDF)
AttributeError: module 'cv2' has no attribute 'TSDF'
```

Reply:
It is just wrong usage. With `class` constant's name is prefixed. It is `VolumeType_TSDF`.

Reply:
got it,  done